test/test_quantization.py

```diff
@@ -259,9 +260,29 @@ def checkQuantized(model):
         checkQuantized(model)
 
         # test one line API
-        model = quantize(ManualQuantModel(), default_eval_fn, calib_data)
+        model = quantize(ManualQuantModel().eval(), default_eval_fn, calib_data)
         checkQuantized(model)
 
+class QuantizationAwareTrainingTest(QuantizationTestCase):
+    def test_manual(self):
+        model = ManualQATModel()
+        model.qconfig = default_qat_qconfig
+
+        prepare(model)
+        self.checkObservers(model)
```

Comment:
Lets call it self.checkFakeQuant here so that it is clear.

Reply:
we need a unified name, this is for both ptq and qat.

Reply:
plan to just call it observer for now